interfaces/apparmor/spec.go

```diff
@@ -122,6 +136,31 @@ func (spec *Specification) AddDeduplicatedSnippet(snippet string) {
 	}
 }
 
+func (spec *Specification) AddParametricSnippet(template, value string) {
```

Comment:
It would be good to unit test this in isolation too.

Reply:
I added a unit test for this now in 0794e580861b43d753a12ac4b1b2999d81f0b82c